packages/graph-explorer/src/connector/openCypher/queries/fetchSchema.ts

```diff
@@ -198,12 +224,12 @@ const fetchSchema = async (
   summary?: GraphSummary
 ): Promise<SchemaResponse> => {
   if (!summary) {
-    const vertices = await fetchVerticesSchema(openCypherFetch) || [];
+    const vertices = (await fetchVerticesSchema(openCypherFetch)) || [];
```

Comment:
Are the extra brackets needed here?

Reply:
These were added automatically by the Prettier rules.